.changeset/quiet-rockets-throw.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/canon': patch
+---
+
+Add new DataTable component.
```

Comment:
Maybe a add a note about updates to the `Table` component too as that one existed already?

Reply:
Done. Thanks for the advice.